app/src/main/java/com/example/puzzlepic/dto/Picture.kt

```diff
@@ -5,8 +5,14 @@ import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
 
 
-data class Picture(@SerializedName("id") var pictureID : String = "", @SerializedName("alt_description") var pictureTitle : String, @SerializedName("created_at") var date : String, var location : String, var notes : String,
-                        var puzzleID : Int = 0, @SerializedName("urls") var urls : UnsplashPhotoUrls ){
+data class Picture(
+    @SerializedName("id") var pictureID : String = "",
+    @SerializedName("alt_description") var pictureTitle : String,
+    @SerializedName("created_at") var date : String,
+    var location : String,
+    var notes : String,
+    var puzzleID : Int = 0,
+    @SerializedName("urls") var urls : UnsplashPhotoUrls ){
```

Comment:
Longer headers should be formated in individual lines. Reference: https://kotlinlang.org/docs/coding-conventions.html#class-headers